pkg/orchestration/docker.go

```diff
@@ -443,6 +445,17 @@ func dockerExec(dockerClient *client.Client, host string, cmds ...string) error
 		return logger.Error("Error reading command output:", err)
 	}
 
+	execInspect, err := dockerClient.ContainerExecInspect(context.Background(), resp.ID)
+	if err != nil {
+		return err
+	}
+	if execInspect.Running {
+		return logger.Errorf("Docker exec process still running with id %d", execInspect.Pid)
+	}
+	if execInspect.ExitCode != 0 {
+		return logger.Errorf("Docker exec process returned non-zero exit code %d", execInspect.ExitCode)
+	}
```

Comment:
we'll want more info right here right? does the execInspect capture other logs or info?

Reply:
It doesn't capture the output, but the `execResp` does (line 440/442) and audius-ctl will point you to the logfile if there's an issue. It's the easiest compromise given the multiple layers of separation trying to run docker commands from a python script inside a docker container on a remote host.